src/config/constants.ts

```diff
@@ -37,3 +37,16 @@ export const eserviceNamingBestPracticeLink =
   'https://italia.github.io/pdnd-guida-nomenclatura-eservice/'
 export const keychainGuideLink = `${documentationLink}/manuale-operativo/utilizzare-i-voucher#garanzia-dellintegrita-della-risposta`
 export const keychainSetupGuideLink = `${documentationLink}/manuale-operativo/utilizzare-i-voucher#precondizioni`
+
+export const SH_ESERVICES_ATT_TEMP = ['9b6993ee-60e3-4901-9a32-e6987d690ec4'] //signal hub eservices to hide
+export const SH_ESERVICES_UAT_TEMP = [
+  //signal hub eservices to hide
+  '7ab0a0fc-7d22-4007-b2f3-fddd68fe2f17',
+  'e8c087eb-627b-4488-a9b7-65b70fd1301b',
+  '407edf51-23b5-462b-af6e-128bbaa4d9ff',
+  '3b0fbe47-2e2c-4d8b-9cff-b2381c92d003',
+  '260e45e1-9a61-49d6-8b6d-da0643da68ac',
+  'a2b84a6e-34cf-44ca-85a4-de21fd232668',
+  '6b14c622-dad2-44ea-82bc-2dd4010364d5',
+  '03c34a8a-a79a-4928-9afc-8647eefabdb1',
+]
```

Comment:
I would make this more flexible:

```typescript
export const SH_ESERVICES_TO_HIDE: Partial<Record<PagoPAEnvVars['STAGE'], Array<string>>> = {
  ATT: ['9b6993ee-60e3-4901-9a32-e6987d690ec4'],
  UAT: [
    '7ab0a0fc-7d22-4007-b2f3-fddd68fe2f17',
    'e8c087eb-627b-4488-a9b7-65b70fd1301b',
    '407edf51-23b5-462b-af6e-128bbaa4d9ff',
    '3b0fbe47-2e2c-4d8b-9cff-b2381c92d003',
    '260e45e1-9a61-49d6-8b6d-da0643da68ac',
    'a2b84a6e-34cf-44ca-85a4-de21fd232668',
    '6b14c622-dad2-44ea-82bc-2dd4010364d5',
    '03c34a8a-a79a-4928-9afc-8647eefabdb1',
  ],
}
```

This would also semplify the logic that uses those